timer.js

```diff
@@ -0,0 +1,13 @@
+let now = 30;
+
+function timer() {
+
+    setTimeout(() => {
+        if (now > 0) {
+            timer();
+        }
+        document.querySelector('#game_timer').textContent = "Timer: " + now;
+        now--;
+
+    }, 1000);
+}
```

Comment:
Issue is that the timer is actually 31s long and starts a second after click, this along with a change on the main.js file should fix that. let now = 30;

`function timer() {

    setTimeout(() => {
        if (now > 0) {
            timer();
        } else {
            return;
        }
        now--;
        document.querySelector('#game_timer').textContent = "Timer: " + now;

    }, 1000);
}
`